### PR TITLE
Create #2089_Find_Target_Indices_After_Sorting_Array.py

### LINKED ISSUE #5: #2089_Find_Target_Indices_After_Sorting_Array
# Summary
add 2089. Find Target Indices After Sorting Array

# Goal
add 2089. Find Target Indices After Sorting Array

# Todo
- [ ] add 2089. Find Target Indices After Sorting Array
- [ ] time complexity: O(n*logn)
- [ ] space complexity: O(1)

# Deadline
/

# Parent issue
# References
None

# Notes
None

### PR DESCRIPTION
**Pull request by ashizawaN**
Closes #5

### CODE REVIEW

File: #2089_Find_Target_Indices_After_Sorting_Array.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Solution:
+    def targetIndices(self, nums: List[int], target: int) -> List[int]:
+        nums.sort()
+        x=[]
```
Comment (nayuta-ai) [inline #2089_Find_Target_Indices_After_Sorting_Array.py:4]: x = []

File: #2089_Find_Target_Indices_After_Sorting_Array.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        
+        for i in range(len(nums)):
+            if nums[i]==target:
```
Comment (nayuta-ai) [inline #2089_Find_Target_Indices_After_Sorting_Array.py:7]: if nums[i] == target:

**Review by nayuta-ai — CHANGES_REQUESTED** [review]

File: #2089_Find_Target_Indices_After_Sorting_Array.py (hunk 1 above)
Comment (ashizawaN) [reply #2089_Find_Target_Indices_After_Sorting_Array.py:4]: @nayuta-ai 
I wrote "[]" in this code. However, when I type with LeetCode or VSCode, it looks like there is a space.

File: #2089_Find_Target_Indices_After_Sorting_Array.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class Solution:
```
Comment (nayuta-ai) [inline #2089_Find_Target_Indices_After_Sorting_Array.py:1]: https://github.com/ashizawaN/LeetCode/pull/2/files#r995428337

**Review by nayuta-ai — CHANGES_REQUESTED** [review]

**Review by nayuta-ai — APPROVED** [review]

**Pull request merged**